CandidateExercise/ConfigurationExercise/Config.cs

```diff
@@ -165,5 +314,11 @@ public void SaveValues(App app, User user, KVPList kvps)
         {
             throw new NotImplementedException();
         }
+
+        Exception catchException(String message, Exception ex, User user = null)
+        {
+            _log.Error(message + user != null ? " for User " + user : String.Empty + " : " + ex.Message);
```

Comment:
Can you use string interpolation for strings comprised of expressions?
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated
The interpolated string format is more readable, and provides some formatting control for later.